WorkWithFiles/features/CucumberProject/feature/sample.feature

```diff
@@ -0,0 +1,33 @@
+Feature: files
+
+  Scenario Outline: Rename and delete a file
+    Given I have <file>
+    When I  <action> a file
+    Then I display the result of <action>
+    Examples:
+      |file         | action   |
+      | a file      | renaming |
+      | not a file  | renaming |
+      | a file      | deleting |
+      | not a file  | deleting |
+
+  Scenario Outline: Create a file
+    When I creates file with <access> right access
+    Then I display the result of creating
```

Comment:
It would be better to write the following step:
Then file with <access> right is created
instead of 'Then I display the result of creating'